### DISCUSSION TITLE
How to deploy a .NET Maui app with pythonnet to Android and iOS ?

### DISCUSSION THREAD

**Discussion by csharpernet** [pythonnet/pythonnet#2177] [Q&A]
Hi everyone,

I'm able to run locally in my windows 11 machine a .NET Maui simple app that has the pythonnet nuget package. I can successfully get data from a python function from my C# code.

Note that I do have a python env local in my windows machine and in my C# code I have:

`Runtime.PythonDLL = @"C:\Users\netland\AppData\Local\Programs\Python\Python311\python311.dll";`

Now, my questions are: how to recreate the same env and its dependencies when I deploy the app to both **iOS and Android** platforms?

What will be the path for **Runtime.PythonDLL** ? How does it work there?

Thanks for your guidance!

-Nuno

**Comment by m7mdhka** [comment]
Up.

**Comment by IcyVizi** [comment]
Any progress on this matter?
